caesars-palace/styles/styles.css

```diff
@@ -295,6 +295,27 @@ main .section {
   padding: 0 var(--layout-size-m);
 }
 
+main .section.has-centered-text > div > p {
```

Comment:
Adds some smaller CSS classes to use in metadata section to control text-alignment and text-color